style.css

```diff
@@ -0,0 +1,262 @@
+  color: #acb7c3;
+}
+
+.profile-container h1 {
+  font-family: Roboto, sans-serif;
+  font-size: 48px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 72px;
+  letter-spacing: 0.3700000047683716px;
+  text-align: center;
+  color: #7d73fc;
+}
+
+.profile-container h2 {
+  font-family: Roboto, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #edecff;
+}
+
+.profile-container p {
+  font-family: Roboto, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: center;
+  color: #c8c8d4;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-top: 40px;
+}
+
+.skill-stack {
+  height: 462.1505432128906px;
+  width: 303px;
+  left: 12px;
+  top: 235px;
+  border-radius: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.skill-stack h3 {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: left;
+  color: #9490ff;
+}
+
+li {
+  list-style: none;
+}
+
+.skill-stack li {
+  font-family: Roboto, sans-serif;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+.social-media {
+  height: 104px;
+  width: 200px;
+  left: 87.5px;
+  top: 1092.1505126953125px;
+  border-radius: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.social-media-icon {
+  height: 40px;
+  width: 200px;
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: #fff;
+  padding-top: 48px;
+}
+
+.experience-title {
+  padding-top: 83.85px;
+}
+
+.experience-title h2 {
+  font-family: Roboto, sans-serif;
+  font-size: 40px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #9490ff;
+}
+
+.experience-container > img {
+  height: 236px;
+  width: 327px;
+  left: 0;
+  top: 0;
+  border-radius: 0;
+  padding-top: 48px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.experience-container > h3 {
+  font-family: Roboto, sans-serif;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  padding-left: 24px;
+  color: #fff;
+}
+
+.experience-container p {
+  font-family: Inter, sans-serif;
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  letter-spacing: 0;
+  text-align: left;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.stack-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 40px;
+  width: 312px;
+  left: 0;
+  top: 357px;
+  border-radius: 0;
+  padding-top: 24px;
+  align-items: center;
+}
+
+.stack-container li {
```

Comment:
- [ ] Kindly fix the styles of these list items to match the given design.

### **Your design**
![Screenshot2](https://user-images.githubusercontent.com/47625407/121765037-7bcc4300-cb48-11eb-897b-006d8f94d74d.png)
### **Original design**
![Screenshot3](https://user-images.githubusercontent.com/47625407/121765041-871f6e80-cb48-11eb-9f48-ac58b8daed7b.png)